### PR TITLE
Update product documentation to better convey the value prop

### PR DESCRIPTION
**Pull request by jayantk**
The docs page for each product needs to explain to the user what the value proposition of each product is. I updated price feeds / entropy so the value prop is (hopefully) clear.

### CODE REVIEW

File: pages/price-feeds/use-real-time-data/evm.mdx [hunk 1]
```diff
@@ -70,7 +70,7 @@ Your contract should do the following four steps:
 
 1. Instantiate the `IPyth` interface from the Solidity SDK using the price feeds contract address.
 1. Call `IPyth.getUpdateFee` to calculate the fee charged by Pyth to update the price.
-1. Call `IPyth.updatePriceFeeds` to update the price.
+1. Call `IPyth.updatePriceFeeds` to update the price, paying the fee calculated in the previous step.
```
Comment (jayantk) [inline pages/price-feeds/use-real-time-data/evm.mdx:73]: drive-by clarification

**Review by aditya520 — APPROVED** [review]
